docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,75 +4,370 @@ title: "Red Hat OpenShift"
+
+:::
+
+  </TabItem>
+  <TabItem value="no-ingress" label="No Ingress">
+If you do not have a domain name or do not intend to use one for your Camunda 8 deployment, external access to Camunda 8 web endpoints from outside the OpenShift cluster will not be possible.
+
+However, you can use `kubectl port-forward` to access the Camunda platform without a domain name or Ingress configuration. For more information, refer to the [kubectl port-forward documentation](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_port-forward/).
+
+To make this work, you will need to configure the deployment to reference `localhost` with the forwarded port. Update your `values.yml` file with the following:
+
+```yaml reference
+https://github.com/camunda/camunda-deployment-references/blob/feature/openshift-ra-standard/aws/rosa-hcp/camunda-versions/8.7/procedure/install/helm-values/no-domain.yml
+```
+
+  </TabItem>
+</Tabs>
+
+#### Configuring the Security Context Constraints
+
+Depending on your OpenShift cluster's Security Context Constraints (SCCs) configuration, the deployment process may vary.
 By default, OpenShift employs more restrictive SCCs. The Helm chart must assign `null` to the user running all components and dependencies.
+
+<Tabs queryString="current-scc">
+<TabItem value="w-scc" label="Restrictive SCCs" default>
+
 The `global.compatibility.openshift.adaptSecurityContext` variable in your values.yaml can be used to set the following possible values:
 
 - `force`: The `runAsUser` and `fsGroup` values will be null in all components.
 - `disabled`: The `runAsUser` and `fsGroup` values will not be modified (default).
 
-To deploy Camunda 8 on OpenShift:
-
-1. Install [Helm and other CLI tools](/self-managed/setup/install.md#prerequisites).
-2. Install the [Camunda Helm chart repository](/self-managed/setup/install.md#helm-repository).
-3. Set `global.compatibility.openshift.adaptSecurityContext` to `force`
-
-```shell
-helm install camunda camunda/camunda-platform --skip-crds \
-  --set global.compatibility.openshift.adaptSecurityContext=force
+```hcl reference
+https://github.com/camunda/camunda-deployment-references/blob/feature/openshift-ra-standard/aws/rosa-hcp/camunda-versions/8.7/procedure/install/helm-values/scc.yml
 ```
 
 </TabItem>
 <TabItem value="no-scc" label="Permissive SCCs">
 
-### With permissive SCCs
-
 To use permissive SCCs, simply install the charts as they are. Follow the [general Helm deployment guide](/self-managed/setup/install.md).
 
+```hcl reference
```

Comment:
learned the hard way about the difference ^^, wish there was more Helm chart feedback as it initially looks successful but then nothing is deployed.